docs/kratos/emails-sms/20_account-recovery-password-reset.mdx

```diff
@@ -3,8 +3,7 @@ id: account-recovery-password-reset
 title: Setting up account recovery and password reset
```

Comment:
Author's comment - I haven't really done anything to this doc aside from moving it to this category. I KINDA feel this is the place for this content but I'm not 100% sure. 

If the placement is okay then we can work on it in the near future.

Reply:
Same as above: https://github.com/ory/docs/pull/946#discussion_r941448918, 
you might use something like the r`evoke_active_sessions` hook, but I think we already have something in console as well for it - not sure, need to check.

Reply:
👍🏻

Reply:
moved to self-hosted